backend/interactive/mixplay.js

```diff
@@ -117,6 +118,10 @@ function addControlHandlers(controls) {
             logger.debug(`Control event "${event}" for control "${inputData.controlID}" in scene "${sceneId}"`);
 
             controlManager.handleInput(event, sceneId, inputEvent, participant);
+
+            if (participant.anonymous !== true) {
```

Comment:
these sorts of conditions can be shortened to `if (!participant.anonymous)` since we can trust that property will always be a Boolean.